docs/core/tools/dotnet-tool-uninstall.md

```diff
@@ -0,0 +1,65 @@
+---
+title: dotnet tool uninstall command - .NET Core CLI
+description: The dotnet tool uninstall command uninstalls the specified .NET Core Global Tool from your machine.
+author: mairaw
+ms.author: mairaw
+ms.date: 04/25/2018
+ms.topic: article
+ms.prod: .net-core
+ms.technology: dotnet-cli
+ms.workload: 
+  - dotnetcore
+---
+# dotnet tool uninstall
+
+[!INCLUDE [topic-appliesto-net-core-2-1plus.md](../../../includes/topic-appliesto-net-core-2-1plus.md)]
+
+## Name
+
+`dotnet tool uninstall` - Uninstalls the specified [.NET Core Global Tool](global-tools.md) from your machine.
+
+## Synopsis
+
+```
+dotnet tool uninstall <PACKAGE_NAME> <-g|--global>
+dotnet tool uninstall <PACKAGE_NAME> <--tool-path>
+dotnet tool uninstall [-h|--help]
+```
+
+## Description
+
+The `dotnet tool uninstall` command provides a way for you to uninstall .NET Core Global Tools from your machine. To use the command, you either have to specify that you want to remove a user-wide tool using the `--global` option or specify a path to where the tool is installed using the `--tool-path` option.
+
+## Arguments
+
+`PACKAGE_NAME`
+
+Name/ID of the NuGet package that contains the .NET Core Global tool to uninstall. You can find the package name using the [dotnet tool list](dotnet-tool-list.md) command.
+
+## Options
+
+`-g|--global`
+
+Specifies that the tool to be removed is from a user-wide installation. Can't be combined with the `--tool-path` option. If you don't specify this option, you must specify the `--tool-path` option.
+
+`-h|--help`
+
+Prints out a short help for the command.
+
+`--tool-path <PATH>`
+
+Specifies the location where to uninstall the global tool. PATH can be absolute or relative. Can't be combined with the `--global` option. If you don't specify this option, you must specify the `--global` option.
+
```

Comment:
We should probably add a warning about uninstall not working on Preview 1 tools. Let's get the other warning right and then see what should go here. Similar warning should probably go in update.

Reply:
Given that we're only shipping these docs now and we don't usually cover preview issues on those unless it's the current functionality, I'm not going to address this comment. But please let me know if you still feel this is needed.